templates/pages/core/get-started/developer-setup.md

```diff
@@ -1,74 +1,56 @@
-----
+---
 Title: Developer setup
-Description:
-----
-
-#Developer setup
-Introduction copy...
-
-##Prerequisites
-Introduction copy...
-* list 1
-* list 2
-* list 3
-
-##Board 1 - Hardware and software requirements
-Introduction copy...
- 1. item1
- 2. item2
- 3. item3
- 4. item4
-
-Summary steps link >
-
-##Board 2 - Hardware and software requirements
-Introduction copy...
- 1. item1
- 2. item2
- 3. item3
- 4. item4
-
-Summary steps link >
-
-##Board 3 - Hardware and software requirements
-Introduction copy...
- 1. item1
- 2. item2
- 3. item3
- 4. item4
-
-Summary steps link >
-
-##Board 4 - Hardware and software requirements
-Introduction copy...
- 1. item1
- 2. item2
- 3. item3
- 4. item4
-
-Summary steps link >
-
-##Board 5 - Hardware and software requirements
-Introduction copy...
- 1. item1
- 2. item2
- 3. item3
- 4. item4
-
-Summary steps link >
-
-##Board 6 - Hardware and software requirements
-Introduction copy...
- 1. item1
- 2. item2
- 3. item3
- 4. item4
-
-Summary steps link >
-
-##Summary steps
-Introduction copy
- 1. item1
- 2. item2
- 3. item3
- 4. item4
+Description: These steps will walk you through installing a developer environment on an Ubuntu Core device.
+---
+
+# Developer setup
+
+Installing a classic Ubuntu environment is required to develop and build snaps directly on your Ubuntu Core device.
+
+## Developing on target
+
+You can use any Ubuntu 16.04 LTS workstation to develop a snap using [snapcraft](http://snapcraft.io/docs/build-snaps/), then build for another architecture [using Launchpad or a chroot setup](http://snapcraft.io/docs/build-snaps/build-for-another-arch), but developing on the target device itself is convenient when testing device specific features and lets you iterate faster.
+
+### Installing and using developer tools
+
+Ubuntu Core provides a read-only file system, that doesn't let you install deb packages. Installing the "classic" snap creates a classic Ubuntu environment for you to use on the device.
```

Comment:
; instead of , => Ubuntu Core provides a read-only file system; that doesn't let you install deb packages